fastplotlib/graphics/selectors/_linear.py

```diff
@@ -278,9 +288,9 @@ def make_ipywidget_slider(self, kind: str = "IntSlider", **kwargs):
         cls = getattr(ipywidgets, kind)
 
         slider = cls(
-            min=self.selection.limits[0],
-            max=self.selection.limits[1],
-            value=int(self.selection()),
+            min=self._selection.limits[0],
+            max=self._selection.limits[1],
+            value=int(self._selection()),
```

Comment:
once `_selection` is defined in the construction just access it using the property.